sbol2/config.py

```diff
@@ -98,6 +99,21 @@ class Config:
     through the setOptions and getOptions methods.
     """
 
+    # Dictionary which holds builder functions for SBOL objects and which can
+    # be extended with extension classes. The dictionary is populated in
+    # document.py
+    SBOL_DATA_MODEL_REGISTER = {}
```

Comment:
What is the advantage of populating `SBOL_DATA_MODEL_REGISTER` in document.py?

Reply:
See previous comment.  It's the easiest way to avoid circular imports without rethinking all the imports.